### PR TITLE
repair wrong slovenian translation

### PR DESCRIPTION
**Pull request by krajniks**
There is a small translation error. This commit fixes it.

### CODE REVIEW

**Comment by JadranR** [comment]
If you are already repairing, fix it properly, please.

File: Translations/WinMerge/Slovenian.po [hunk 1]
```diff
@@ -596,7 +596,7 @@ msgstr "Shrani sredn&jo kot..."
 
 #, c-format
 msgid "Save &Right As..."
-msgstr "Shrani l&evo kot..."
```
Comment (JadranR) [inline Translations/WinMerge/Slovenian.po:599]: msgstr "Shrani &desno kot..."

**Comment by krajniks** [comment]
Thanks @JadranR. I will incorporate your suggestions.

**Comment by krajniks** [comment]
> Please, whats mean this "Nima vpliva popravke"????
> Anyone with 5 minutes would like to correct the translation, what about homework?
> Sorry, But it's not normal :-(

Thanks for the  check @JadranR. Just want to make this product better. There is more things in the translation that needs to be corrected/enhanced, but are outside my knowledge scope.

**Comment by krajniks** [comment]
You are correct about Scripts. That was totaly my fault, thanks for pointing it out.

"Vie&w Line Differences" is currently "Prikaži razl&ik vrstic". All other tags in the same menu popup use "Prikaz".
image

3-way compare is probably ment as tehnical term used in development just like as 3-way merge is used when mergin files in git and means _združevanje treh_, thus my suggestion of _3-smerna primerjava_.  It is used in menu Show (Prikaz) when comparing 3 folders, if comparing only two, all options under it are grayed out.
image

anyway, as you have already incorporated all of my suggestions, this pull request is no longer needed and I will delete it.

**Review by JadranR — APPROVED** [review]

**Comment by sdottaka** [comment]
@JadranR 

I changed the term "Ignore Substitutions" to "Substitution Filters".
Sorry for the need to change some of the things you translated.

**Comment by JadranR** [comment]
Hi,

NOT PROBLEMS.
Here is updated Slovenian translation at 19.31 (Time) .

With respect,
Jadran
[WinMerge-2.16.9-x64 Updated 27.01.2021.zip](https://github.com/WinMerge/winmerge/files/5882301/WinMerge-2.16.9-x64.Updated.27.01.2021.zip)

**Comment by sdottaka** [comment]
Thank you!

**Comment by JadranR** [comment]
Hi,
Here is Last Slovenian translation after release WinMerge-2.16.10.

Best Regards,
Jadran
[Slovenian WinMerge-2.16.10-x64 30.01.2021.zip](https://github.com/WinMerge/winmerge/files/5897724/Slovenian.WinMerge-2.16.10-x64.30.01.2021.zip)

**Comment by sdottaka** [comment]
> 
> 
> Hi,
> Here is Last Slovenian translation after release WinMerge-2.16.10.
> 
> Best Regards,
> Jadran
> [Slovenian WinMerge-2.16.10-x64 30.01.2021.zip](https://github.com/WinMerge/winmerge/files/5897724/Slovenian.WinMerge-2.16.10-x64.30.01.2021.zip)

Committed at d1eb2cd

Thank you!

**Pull request closed without merging**